modules/test/playwright/tests/commerce/commerceChannelDefaults.spec.ts

```diff
@@ -341,3 +341,98 @@ test('LPD-26142 A Sales Agent can manage channel defaults', async ({
 		await performLogin(page, 'test');
 	}
 });
+
+test('LPD-30466 Verify users without edit permission cannot click on channel name link', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	page,
+}) => {
+	const companyId = await page.evaluate(() => {
+		return Liferay.ThemeDisplay.getCompanyId();
+	});
+
+	const site = await apiHelpers.headlessSite.createSite({
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: site.id, type: 'site'});
+
+	const channel = await apiHelpers.headlessCommerceAdminChannel.postChannel({
+		siteGroupId: site.id,
+	});
+
+	apiHelpers.data.push({id: channel.id, type: 'channel'});
+
```

Comment:
I think this is not needed because the postChannel already add it (please verify)